QEfficient/utils/_utils.py

```diff
@@ -42,9 +42,8 @@ def hf_download(
     if cache_dir is not None:
```

Comment:
can you change the function definition to take max_retries as input and set it's default value to `constants.MAX_RETRIES`

Reply:
@ochougul this can be a configurable value. so it would be better if we can fetch from a file. If we need to make the max_retries as input we need to provide a way for user to set it.